modules/kubernetes-addons/main.tf

```diff
@@ -530,3 +530,14 @@ module "kubecost" {
   manage_via_gitops = var.argocd_manage_add_ons
   addon_context     = local.addon_context
 }
+
+
+module "kyverno" {
+  count                        = var.enable_kyverno ? 1 : 0
+  source                       = "./kyverno"
+  addon_context                = local.addon_context
+  kyverno_helm_config          = var.kyverno_helm_config
```

Comment:
What I think my colleague was getting at with his review was that - currently you are making users opt in to all of these Kyverno addons or none and instead we should give them the option to enable/disable each one independently. For example, most users might want to enable Kyverno and the baseline but not the UI

Reply:
Thanks @bryantbiggs for the feedback, let me make the changes and push

Reply:
I have incorporated the feedback in the latest commit. Please check.